src/components/GameListItem.vue

```diff
@@ -109,12 +109,12 @@ export default {
       return `${this.numPlayers} / 2`;
     },
     joinButtonText() {
-      return this.isRanked ? 'Play Ranked' : 'Play';
+      return this.isRanked ? 'Join Ranked' : 'Join Casual';
     },
     buttonAttrs() {
       return {
-        color: 'primary',
-        rounded: true,
+        color: 'surface-1',
+        rounded: false,
```

Comment:
I don't believe this is needed, I think you can just omit the rounded attr

Reply:
Thanks for suggestions i will implement it.

Reply:
No doubt man! Keep up the awesome work!